lib/ansible/modules/cloud/digital_ocean/digital_ocean.py

```diff
@@ -307,7 +307,7 @@ def find(cls, id=None, name=None):
     @classmethod
     def list_all(cls):
         json = cls.manager.all_active_droplets()
-        return map(cls, json)
+        return list(map(cls, json))
```

Comment:
Not a blocker but you could also do it like this:
``` python
return [cls(j) for j in json]
```
list comprehensions and generator expressions are a more pythonic idiom in general... I only mention it here, though, because it's nicer to encapsulate that in one idiom than into a nested function call (calling both map and list).